src/subscription_manager/managercli.py

```diff
@@ -701,12 +701,12 @@ def _do_command(self):
             self.entcertlib.update()
 
             log.info("Refreshed local data")
-            print (_("All local data refreshed"))
-        except connection.RestlibException, re:
-            log.error(re)
-            system_exit(os.EX_SOFTWARE, re.msg)
-        except Exception, e:
-            handle_exception(_("Unable to perform refresh due to the following exception: %s") % e, e)
+            print(_("All local data refreshed"))
```

Comment:
Since you are not making changes related to the fix here, please don't include this as part of the PR. The general rule we should stick to is: if you're already changing the line, then go ahead and make syntax/style improvements, otherwise do it in another PR.

I do like the changes though. Please put these changes in a separate PR, and coordinate w/ @awood since these are related to Python-3 changes (which Al is looking at).